_guides/getting_started.md

```diff
@@ -57,64 +86,139 @@ best constant = 0.500000
+
+This diagnostic shows that the number of hash function bits is 18 (more than enough for this example). Use `-b bits` to adjust the number of bits for a hash function.
+
+### Learning rate
+
+The following code sample shows the learning rate:
 
 ```
 learning rate = 0.5
 ```
-The default learning rate is 0.5 which we found to be a good default with the current default update (`--normalized --invariant --adaptive`). If the data is noisy you'll need a larger data-set and/or multiple passes to predict well. On these larger data-sets, our learning rate will by default decay towards 0 as we run through examples. You can adjust the learning rate up or down with `-l rate`. A higher learning rate will make the model converge faster but a too high learning rate may over-fit and end-up worse on average.
+
+The default learning rate is `0.5` with current default update (`--normalized --invariant --adaptive`). 
+
+If the data is noisy, you need a larger data-set or multiple passes to predict well. For massive data-sets, the learning rate decays to `0` by default. 
+
+**Use** `-l rate` to adjust the learning rate up or down.
+
+.**Note:** A high learning rate makes the model converge faster than a low learning rate, but if you adjust the learning rate too high, you risk over-fit and end-up worse on average.
+
+### Learning rate decay
+
+The following code sample shows the initial time:
 
 ```
 initial_t = 0
 ```
-Learning rates should often decay over time, and this specifies the initial time. You can adjust with `--initial_t time`, although this is rarely necessary these days.
+
+This diagnostic specifies the initial time or duration for learning rate decay. 
+
+**Use** `--initial_t time` to adjust the initial time. 
+
+>**Note:** It is not common to adjust the initial time in Vowpal Wabbit
+
+### Learning rate decay power
+
+The following code sample shows the power of learning rate decay:
 
 ```
 power_t = 0.5
 ```
-This specifies the power on the learning rate decay. You can adjust this `--power_t p` where _p_ is in the range [0,1]. 0 means the learning rate does not decay, which can be helpful when state tracking, while 1 is very aggressive, but plausibly optimal for [IID](https://en.wikipedia.org/wiki/Independent_and_identically_distributed_random_variables) data-sets. 0.5 is a minimax optimal choice. A different way of stating this is: stationary data-sets where the fundamental relation between the input features and target label are not changing over time, should benefit from a high (close to 1.0) `--power_t` while learning against changing conditions, like learning against an adversary who continuously changes the rules-of-the-game, would benefit from low (close to 0) `--power_t` so the learner can react quickly to these changing conditions. For many problems, 0.5, which is the default, seems to work best.
+
+This diagnostic specifies the power of learning rate decay. The VW default is `0.5` and works well for most problems. 
```

Comment:
```suggestion
This diagnostic shows the power on learning rate decay. The VW default is `0.5` and works well for most problems. 
```

The change from "on" to "of" changes the meaning of the expression. It power in the mathematical sense and not metaphorical one.

The diagnostic output only shows what was used.